include/eve/arch/riscv/tags.hpp

```diff
@@ -0,0 +1,115 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/cardinals.hpp>
+#include <eve/arch/cpu/tags.hpp>
+#include <eve/arch/riscv/predef.hpp>
+#include <eve/concept/scalar.hpp>
+#include <eve/detail/meta.hpp>
+
+#include <bit>
+
+namespace eve
+{
+//================================================================================================
+// ABI tags for all RISCV bits SIMD registers
+//================================================================================================
+template<std::size_t Size> struct rvv_abi_
+{
+  static constexpr std::size_t MaxLmul         = 8;
```

Comment:
> From what I can see it's like the hardware will start using more registers

Yep, that is correct: when lmul > 1 in operation more physical vector registers are used.

> Is it beneficial in any way, over just unrolling n times?

For now I just optimized this in terms of number of executed instructions. I suppose by default it is expected that performing one vector operation with 8 registers is faster that performing 8 vector operations with 1 vector register, otherwise I see no reasons for adding this ability in ISA. Though this value seems to be hardware-dependent.